spec/services/atmosphere/optimizer_spec.rb

```diff
@@ -145,6 +145,25 @@
     end
   end
 
+  context 'selection is quick' do
+    it 'selects template, flavor and tenant quickly' do
+      vmts = []
+      100.times do
+        atype = create(:appliance_type)
+        t = create(:openstack_with_flavors, funds: [fund])
+        vmts << create(
+          :virtual_machine_template,
+          appliance_type: atype,
+          tenants: [t]
+        )
+      end
+      t1 = Time.now.to_f
+      subject.select_tmpl_and_flavor_and_tenant(vmts)
+      t2 = Time.now.to_f
+      expect(t2 - t1).to be < 5
```

Comment:
Please add error message to know what is wrong when this check fails (https://www.relishapp.com/rspec/rspec-expectations/docs/customized-message).